SinaiStCatherine/SinaiEt004.xml

```diff
@@ -471,32 +472,34 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 <list>
                                   <item xml:id="a1">
                                     <locus target="#1v"/>
-                                    <desc type="Unclear">Unclear note, seems to indicated the dimensions of the body of Adam.</desc>
+                                    <desc type="Unclear">An note indicating the great stature of <persName ref="PRS1386Adam">Adam</persName></desc>
```

Comment:
If physical dimensions are meant, I think "dimensions of the body" is a bit more unambiguous

Reply:
rephrased.